keyboards/ciaanh/kanagawa/ansi/keymaps/via/keymap.c

```diff
@@ -0,0 +1,112 @@
+*   ,-------------------------------------------------------------------------------------------------------------------------------------------.    ,---------.
+*   |    `    |    1    |    2    |    3    |    4    |    5    |    6    |    7    |    8    |    9    |    0    |    -    |    =    |Backspace|    |  Delete |
+*   |---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|    |---------|
+*   |   Tab   |    Q    |    W    |    E    |    R    |    T    |    Y    |    U    |    I    |    O    |    P    |    [    |    ]    |    \    |    |   HOME  |
+*   |---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'    |---------|
+*   |  CapsL  |    A    |    S    |    D    |    F    |    G    |    H    |    J    |    K    |    L    |    ;    |    '    |      ENTER        |    |   END   |
+*   |---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'---------'    `---------'
+*   |   LShift    | --- |    Z    |    X    |    C    |    V    |    B    |    N    |    M    |    ,    |    .    |    /    | RShift  |  ,---------.
+*   |---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'  |    UP   |
+*   |  LCtrl  |   LGUI  |   LAlt  |  LOWER  |           ,---------.     ,---------.           |   RAlt  |  RAISE  |  RCtrl  |  ,---------|---------|---------.
+*   |---------+---------+---------+---------+           |  Space  |     |  Space  |           +---------+---------+---------'  |  LEFT   |   DOWN  |  RIGHT  |
+*                                                       `---------'     `---------'                                            `---------+---------+---------'  
+*/
+    [_BASE] = LAYOUT(  
+         KC_ESC,   KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,   KC_F6,   KC_F7,   KC_F8,     KC_F9,  KC_F10,  KC_F11,  KC_F12,           KC_MPLY,
+      
+         KC_GRV,    KC_1,       KC_2,       KC_3,       KC_4,       KC_5,    KC_6,    KC_7,    KC_8,      KC_9,    KC_0, KC_MINS,  KC_EQL, KC_BSPC,   KC_DEL,
+         KC_TAB,    KC_Q,       KC_W,       KC_E,       KC_R,       KC_T,    KC_Y,    KC_U,    KC_I,      KC_O,    KC_P, KC_LBRC, KC_RBRC, KC_BSLS,  KC_HOME,
+        KC_CAPS,    KC_A,       KC_S,       KC_D,       KC_F,       KC_G,    KC_H,    KC_J,    KC_K,      KC_L, KC_SCLN, KC_QUOT,  KC_ENT,            KC_END,
+        KC_LSFT, XXXXXXX,       KC_Z,       KC_X,       KC_C,       KC_V,    KC_B,    KC_N,    KC_M,   KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,    KC_UP,
+        KC_LCTL, KC_LGUI,    KC_LALT, MO(_LOWER),        KC_SPC,        KC_SPC,             KC_RALT,MO(_RAISE), KC_RCTL,           KC_LEFT, KC_DOWN, KC_RGHT
```

Comment:
Via enables the tri layer stuff by default, so you can use them by just adding: 
```suggestion
        KC_LCTL, KC_LGUI,    KC_LALT,  TL_LOWR,        KC_SPC,        KC_SPC,             KC_RALT, TL_UPPR, KC_RCTL,           KC_LEFT, KC_DOWN, KC_RGHT
```